image-swipe.ios.ts

```diff
@@ -132,27 +132,29 @@ export class ImageSwipe extends ImageSwipeBase {
 
     private _loadCurrentPage(page: number) {
         const scrollView: UIScrollView = this.nativeViewProtected;
-        const pageWidth = scrollView.frame.size.width;
+        if (scrollView) {
```

Comment:
Let's just reverse the logic and short circuit execution with a `return` here so we don't have to surround the whole function body.

Reply:
I agree